### DISCUSSION TITLE
Internews budget for the project

### DISCUSSION THREAD

**Discussion by abitrolly** [bleachbit/bleachbit#1479] [Ideas]
@az0 there is funding available for security related open source teams for planning their sustainability plan, and I believe BleachBit fits the description. The deadline is 28 of April, and all details are here https://internews.org/call-for-expressions-of-interest-strategic-planning-for-open-source-software-sustainability-project-sustain/

**Comment by az0** [comment]
@abitrolly 
Thank you for letting me know, but sorry I missed the deadline. Hmm, interesting they are pushing projects to adopt the Clean Insights SDK

**Reply by abitrolly** [reply]
That SDK is shady as hell. :D "Spy on users without spying on them." It allows to avoid sharing user data with 3rd party services, but only if you are tech savvy enough to run own server. Which is this thing - https://matomo.org/changelog/matomo-5-0-0/ - released yesterday. Not sure if @inson1 appearance is an effect of that change.

And then nothing stops you from selling this data. More over, nothing stops SDK authors from inserting some backdoors in updates. So insisting to inject this dependency into Open Source projects only means that they want to propose you to sell this data at some point. Or maybe the grant is the selling contract - is there a term that OS projects need to provide "the proof" that the users are using the projects? Even without direct access to user data, the packets that SDK says most likely could be fingerprinted. So it is definitely a bad move to push for such things.

**Reply by inson1** [reply]
@abitrolly interesting coincidence lol, but I dont know anything about that, this project missed deadline for that funding anyway, which is solving that problem, right?

**Reply by inson1** [reply]
@abitrolly btw you pushed for that funding

**Reply by abitrolly** [reply]
Iep. That's why I am getting notifications. But the round isn't "closed" - the correct status is "missed the deadline".  :D

**Reply by inson1** [reply]
@abitrolly Isnt that almost same thing? Plus it has been half year and nobody cared.

**Reply by abitrolly** [reply]
No. "Closing round" means "getting money" in startup terms. At least that's how my fellow friends are talking big.

**Comment by inson1** [comment]
@abitrolly  May you close it? :)

**Reply by abitrolly** [reply]
@inson1 why do you insist on closing it? :D

**Reply by inson1** [reply]
@abitrolly its after deadline, right? My only reason :)

**Reply by abitrolly** [reply]
Yea, well, normally the discussions don't have to be closed. It is not tracker after all.

**Reply by inson1** [reply]
@abitrolly Was it active disscussion? I like when solved things are closed tbh. So Its nice and clean.

I dont like when I have to go through outdated things to find something active or still relevant.
